modules/blobs/data_mappings.bal

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/lang.array as arrlib;
+import ballerina/http;
+import ballerina/jsonutils;
+
+# Converts Container JSON into Container Type.
+# 
+# + containerJsonObject - json container object
+# + return - Returns Container type
+isolated function convertJSONToContainerType(json containerJsonObject) returns Container|error {
+    Container container = check containerJsonObject.cloneWithType(Container);
+    return container;
+}
+
+# Converts Blob JSON into Blob Type.
+# 
+# + blobJsonObject - json blob object
+# + return - Returns Blob type
+isolated function convertJSONToBlobType(json blobJsonObject) returns Blob|error {
+    Blob blob = check blobJsonObject.cloneWithType(Blob);
+    return blob;
+}
+
+# Converts Storage Service Properties JSON into StorageServiceProperties Type.
+# 
+# + storageServicePropertiesJson - json Storage Service Properties object
+# + return - Returns Blob type
+isolated function convertJSONtoStorageServiceProperties(json storageServicePropertiesJson) 
+        returns StorageServiceProperties|error {
+    StorageServiceProperties properties = check storageServicePropertiesJson.cloneWithType(StorageServiceProperties);
+    return properties;
+}
+
+# Creates AccountInformationResult from http response.
+# 
+# + response - validated http response
+# + return - Returns AccountInformation type
+isolated function convertResponseToAccountInformationType(http:Response response) 
+        returns @tainted AccountInformationResult|error {
+    AccountInformationResult accountInformation = {};
+    accountInformation.accountKind = response.getHeader(X_MS_ACCOUNT_KIND);
+    accountInformation.skuName = response.getHeader(X_MS_SKU_NAME);
+    accountInformation.isHNSEnabled = response.getHeader(X_MS_IS_HNS_ENABLED);
+    accountInformation.responseHeaders = getHeaderMapFromResponse(<http:Response>response);
+    return accountInformation;
```

Comment:
AccountInformationResult.skuName is string. But sometimes, get header method can send an error like HeaderNotFoundError. then will it be set to the AccountInformationResult.skuName?

Reply:
Since they are mandatory fields, we won't get that error.